packages/@jsii/java-runtime/package.json

```diff
@@ -25,7 +25,7 @@
   "types": "lib/index.d.ts",
   "scripts": {
     "gen": "bash ./generate.sh",
-    "build": "tsc --build && npm run gen && cd project && mvn -version && mvn -B deploy -D altDeploymentRepository=local::default::file://${PWD}/../maven-repo --settings=user.xml",
+    "build": "tsc --build && npm run gen && cd project && mvn -version && mvn -B deploy -D altDeploymentRepository=local::default::file://${PWD}/../maven-repo --settings=user.xml -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false",
```

Comment:
You probably want to do the same/similar in java-runtime-test?

Reply:
Yep. Done.